include/mbgl/shaders/gl/drawable_line_gradient.hpp

```diff
@@ -0,0 +1,196 @@
+// Generated code, do not modify this file!
```

Comment:
I think `drawable_line_*.hpp` files should not be part of this PR.

Reply:
done